### PR TITLE
fix(parsers): handle N/A duration values to prevent NaN

### PR DESCRIPTION
**Pull request by mekwall**
This pull request improves the `timerToSecs` utility function and its associated test coverage, while also enhancing the `parseInfo` function to handle edge cases like "N/A" duration values. The changes focus on better error handling, code robustness, and expanded test scenarios.

### Enhancements to `timerToSecs` utility function:

* [`src/utils/timerToSecs.ts`](diffhunk://#diff-6ab43e0e1489301b57445638535e463902b1b7dd0d41a56b6e645486fba72786R6-R32): Added comprehensive error handling for invalid inputs, including empty strings, "N/A", non-numeric values, and incorrect formats. Improved input validation and error messages for edge cases.

* [`src/utils/__tests__/timerToSecs.spec.ts`](diffhunk://#diff-b3fde0f018cc56b4f7bb6a280232402bcff6e8d23ed4fe40af842ca270a1562cL4-R81): Expanded test coverage for `timerToSecs` with cases for valid inputs, edge cases, and invalid formats. Tests now include scenarios like large values, decimal seconds, whitespace handling, and various error conditions.

### Improvements to `parseInfo` function:

* [`src/parsers.ts`](diffhunk://#diff-ae1b7b231590dceb8b2fd2a743fcab67b4e2d62bae9ac006b5c1ef2772948e9dR59-R67): Updated the `parseInfo` function to handle "N/A" duration values gracefully by setting `duration` to `undefined` instead of attempting to parse. Refactored duration parsing logic for clarity.

* [`src/__tests__/parsers.spec.ts`](diffhunk://#diff-3678d907cd965420c60a2764884d3b8fe364d096a92a74c50471e0105d9f8b7aR168-R178): Added a test case for "N/A" duration handling in the `parseInfo` function to ensure correct behavior.

Closes https://github.com/mekwall/ffmpeggy/issues/29

### CODE REVIEW

File: src/parsers.ts [hunk 1]
```diff
@@ -56,8 +56,15 @@ export function parseInfo(data: string): FFmpegInfo | undefined {
   if (!matches) {
     return;
   }
+  const durationStr = matches[1];
+  let duration: number | undefined;
+
+  if (durationStr && durationStr.trim() !== "N/A") {
+    duration = timerToSecs(durationStr);
```
[the remark this answers is not in the document]
Comment (mekwall) [reply src/parsers.ts]: Fixed in [4bb20f9](https://github.com/mekwall/ffmpeggy/pull/711/commits/4bb20f981b56bc6534a3be2a9adb8c5da785ca3f)
